spyder/plugins/editor/panels/linenumber.py

```diff
@@ -98,6 +99,14 @@ def draw_pixmap(xleft, ytop, pixmap):
         for top, line_number, block in self.editor.visible_blocks:
             data = block.userData()
             if self._markers_margin and data:
+                if is_cell_header(block):
+                    cell_line_color = QColor(Qt.darkGray)
```

Comment:
Let us here the color defined at https://github.com/spyder-ide/spyder/pull/20509/files#diff-04bc4dbe5c8af45aeb3dc9d7aef7317a8ae81284b422f34c5e6447c233d54324R37